core/test/basin.jl

```diff
@@ -3,6 +3,15 @@ using Ribasim
 import BasicModelInterface as BMI
 using SciMLBase
 
+
+@testset "trivial model" begin
+    toml_path = normpath(@__DIR__, "../../data/trivial/trivial.toml")
+    @test ispath(toml_path)
+    model = Ribasim.run(toml_path)
+    @test model isa Ribasim.Model
+    @test model.integrator.sol.retcode == Ribasim.ReturnCode.Success
```

Comment:
Does it make sense to test results too?
Like "there is no less water in the basin then there was in the beginning?"